lib/src/SortableList/sortable-list.scss

```diff
@@ -52,6 +54,14 @@
     border-radius: 0.25rem;
     border: 1px solid $gray-300;
     padding: 0.25rem 0.75rem;
+    &.on-drag {
+        background-color: $white;
+        position: fixed;
+        top: -$zindex-tooltip;
+        left: -$zindex-tooltip;
```

Comment:
Are you sure the values of top and left should be the z-index values? I don't really get this part.

Reply:
replace variables with value